### PR TITLE
Fix documentation

### PR DESCRIPTION
**Pull request by astha-jain**
**What this PR does / why we need it**:
Fixes the following terms in documentation
`Trouble shooting` -> `Troubleshooting`
`keypair` -> `key pair`

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Specify your PR reviewers**:

/cc @gyliu513 @jichenjc @xunpan 

**Special notes for your reviewer**:

/area provider/ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
None
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
     - [Cluster Deletion](#cluster-deletion)
     - [Create additional worker nodes](#create-additional-worker-nodes)
     - [How to use clusterctl image](#how-to-use-clusterctl-image)
-    - [Trouble shooting](#trouble-shooting)
+    - [Troubleshooting](#troubleshooting)
```
Comment (spzala) [inline README.md:21]: @astha-jain good catch! 
/lgtm

**Comment by spzala** [comment]
@gyliu513 can you please take a look? Thanks!

**Comment by gyliu513** [comment]
/lgtm
/approve

Thanks @astha-jain for the contribution!

**Pull request merged**